packages/admin/cms-admin/src/userPermissions/UserGrid.tsx

```diff
@@ -155,50 +149,16 @@ export const UserPermissionsUserGrid = ({ toolbarAction, rowAction, actionsColum
             align: "right",
             pinned: "right",
             disableExport: true,
-            renderCell: (params) => {
-                const isCurrentUser = params.row.id === currentUser.id;
-                return (
-                    <>
-                        <Tooltip
-                            title={
-                                isCurrentUser ? (
-                                    <FormattedMessage
-                                        id="comet.userPermissions.cannotImpersonateYourself"
-                                        defaultMessage="You can't impersonate yourself"
-                                    />
-                                ) : (
-                                    commonImpersonationMessages.impersonate
-                                )
-                            }
-                        >
-                            {/* span is needed for the tooltip to trigger even if the button is disabled*/}
-                            <span>
-                                <IconButton
-                                    disabled={isCurrentUser && !isImpersonated}
-                                    onClick={() => {
-                                        if (!isCurrentUser) {
-                                            startImpersonation(params.row.id.toString());
-                                        }
-                                        if (isCurrentUser && isImpersonated) {
-                                            stopImpersonation();
-                                        }
-                                    }}
-                                >
-                                    <ImpersonateUser />
-                                </IconButton>
-                            </span>
```

Comment:
Removing the impersonation from the grid seems like a step back from a UX POV. I'd rather always show the button in the grid, check if it's allowed when clicking on it and otherwise showing an error stating "You can't impersonate a more privileged user".

Reply:
IMHO you won't frequently impersonate users requiring this button in the grid

Reply:
> IMHO you won't frequently impersonate users requiring this button in the grid

I've never used the other way. Why would I? That's two (unnecessary) clicks more.